adminapi/request.py

```diff
@@ -91,7 +91,14 @@ def calc_signature(private_key, timestamp, data=None):
     Returns the signature as base64 encoded unicode, ready for transport.
     """
     message = calc_message(timestamp, data)
-    sig = private_key.sign_ssh_data(message.encode())
+    try:
+        sig = private_key.sign_ssh_data(message.encode())
+    except SSHException as e:
+        # There is a possbility, that key is unable to be used for auth
```

Comment:
Typo: possbility -> possibility
Better English: "There is a possibility that this key cannot be used for auth"

Reply:
Thank you